tools/kore-rich-header/main.cpp

```diff
@@ -0,0 +1,80 @@
+#include <kllvm/ast/AST.h>
+#include <kllvm/parser/KOREParser.h>
+
+#include <cstdint>
+#include <iostream>
+
+using namespace kllvm;
+using namespace kllvm::parser;
+
+int main(int argc, char **argv) {
+  char *filename = argv[1];
+  kore_parser parser(filename);
+  ptr<kore_definition> definition = parser.definition();
```

Comment:
Feel free not to given you're the only consumer of this tool @dwightguth, but consider setting up an LLVM command line argument parser:
- Keeps the signature of the tool documented with `-h` flags / usage message
- Makes it easier to add extra features / behaviour in the future if you need to